content/docs/operating/integrations.md

```diff
@@ -49,6 +49,7 @@ data volumes.
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
   * [Splunk](https://github.com/lukemonahan/splunk_modinput_prometheus#prometheus-remote-write): write
   * [TiKV](https://github.com/bragfoo/TiPrometheus): read and write
+  * [Thanos](http://github.com/improbable-eng/thanos): write\*
```

Comment:
The * is to makes a note below, it's not of any use on its own

Reply:
agreed

Reply:
Any preferred info there? (:  Th

Something like:
```
* Thanos has experimental support for sending metrics via remote write. It uses remote read as the main way to get metrics from Prometheus. 
```

Reply:
sgtm

Reply:
The experimentalness of things isn't relevant on this page. I'd mention that it doesn't have remote read, but exposes a Prometheus-compatible HTTP API.

Reply:
> but exposes a Prometheus-compatible HTTP API.

I think this is can confuse users, as most of those solution has this. Also remote-read and `federate` are Prometheus HTTP API as well and Thanos does not support them (: I would stick to simple version:

`Thanos does not expose remote read API, but uses remote read as the main way to get metrics from Prometheus.`

Reply:
Talking about the remote read API will just confuse people who don't know Thanos I think.

Reply:
ok, so bit confused now (: Do we need any `*` then.